### DISCUSSION TITLE
Is Phantom Deeplink Supported in Webapp?

### DISCUSSION THREAD

**Discussion by teodorus-nathaniel** [phantom/docs#283] [General]
I would like my app to be like the one in this [deeplink guide](https://phantom.app/learn/blog/the-complete-guide-to-phantom-deeplinks)
but for the demo, it was using Expo and in webapp, I don't have access to `Linking.createURL`, or can I just use https url for the redirect?

**Comment by adamdelphantom** [comment]
@teodorus-nathaniel Yes, you can just use the same deeplink url in your web app if it is opened by the user outside of the phantom dapp browser.

**Reply by teodorus-nathaniel** [reply]
Thanks for confirming, yeah I tried again and found something wrong with my reproduction, so I can connect properly now 🙏
